tests/Functional/FetchBooleanTest.php

```diff
@@ -6,13 +6,17 @@
 
 use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
 use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
 use PHPUnit\Framework\Attributes\DataProvider;
 
 class FetchBooleanTest extends FunctionalTestCase
 {
     protected function setUp(): void
     {
-        if ($this->connection->getDatabasePlatform() instanceof PostgreSQLPlatform) {
+        if (
+            $this->connection->getDatabasePlatform() instanceof PostgreSQLPlatform
+            && ! TestUtil::isPdoStringifyFetchesEnabled()
```

Comment:
If we stringify everything, we obviously can't expect actual booleans being returned from the database. Makes sense. Below we emit a message on why we skip the test. The message is not kind of odd because we actually do skip the test on Postgres sometimes.

Can we update it accordingly, for the next person who stumbles over this skipped test?

Reply:
updated 👍